examples/SignalingTestApp/SignalingTestApp/SceneDelegate.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SceneDelegate.swift
+//  SignalingTestApp
+//
+//  Created by Gloria Li on 2021-01-11.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        guard let _ = (scene as? UIWindowScene) else { return }
```

Comment:
If this isn't a SwiftUI app, this file should be removed.

Reply:
@glorialimicrosoft we can exclude the test app for now until we are ready to include the signalling changes. We can make a note of the feedback received here so we can address it later